app.js

```diff
@@ -18,8 +18,16 @@ function getRandomQuestion(){
     return randomQuestion;
 }
 
-function checkTheAnswer(answer){
-    if(question.answer === answer){
+function answerToLowerCase(string){
+    var answer = '';
+    for(var char in string){
```

Comment:
no need to do `for` loop. `char` var is of type string. There is no special `char` type in javascript.
So any string has `toLowerCase` method.